avoid "declaration is incompatible with previous" style error message
cfr. http://lists.gnu.org/archive/html/help-octave/2011-10/msg00438.html
by Kenneth Hoste (HPC-UGent)
--- octave-3.8.2/libgnu/stdio.in.h.orig	2015-04-22 20:12:39.436847845 +0200
+++ octave-3.8.2/libgnu/stdio.in.h	2015-04-22 20:15:20.800889451 +0200
@@ -633,7 +633,7 @@
 _GL_FUNCDECL_RPL (getchar, int, (void));
 _GL_CXXALIAS_RPL (getchar, int, (void));
 # else
-_GL_CXXALIAS_SYS (getchar, int, (void));
+//_GL_CXXALIAS_SYS (getchar, int, (void));
 # endif
 _GL_CXXALIASWARN (getchar);
 #endif
@@ -907,7 +907,7 @@
 _GL_FUNCDECL_RPL (putchar, int, (int c));
 _GL_CXXALIAS_RPL (putchar, int, (int c));
 # else
-_GL_CXXALIAS_SYS (putchar, int, (int c));
+//_GL_CXXALIAS_SYS (putchar, int, (int c));
 # endif
 _GL_CXXALIASWARN (putchar);
 #endif
@@ -1261,7 +1261,7 @@
 /* Need to cast, because on Solaris, the second parameter is
                                                           __va_list args
    and GCC's fixincludes did not change this to __gnuc_va_list.  */
-_GL_CXXALIAS_SYS_CAST (vprintf, int, (const char *format, va_list args));
+//_GL_CXXALIAS_SYS_CAST (vprintf, int, (const char *format, va_list args));
 # endif
 _GL_CXXALIASWARN (vprintf);
 #endif
